pyproject.toml

```diff
@@ -11,7 +11,7 @@ skips = ["*/*test.py", "*/test_*.py", "*tests/*.py"]
 branch = true
 
 [tool.coverage.report]
-fail_under = 95
+fail_under = 92
```

Comment:
I think this is because of admin_access_token.py file. If there is a reason not to test it, could it be better to include a #pragma: no cover there with possibly the reason? (https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-coverage)

Reply:
I don't think that there is a lot of added value to test something that is just a few logic around using juju internals

Reply:
From the guide, I think we are still covered:
https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-coverage

But more tests can be added later.

Reply:
Sure, feel free to close this comment, just that the other option, putting "# pragma: no cover" in the files/places you think are not necessary to test, will not change the required test coverage for future PRs.